### PR TITLE
test-configs.yaml: Add and enable LTP-open-posix-timers on devices 

### PR DESCRIPTION
**Pull request by Lakshmipathi**
Add ltp-open-posix timers test definition. Enable ltp-open-posix timers
on lab-collabora devices where tests are passed.

Signed-off-by: lakshmipathi.g <lakshmipathi.ganapathi@collabora.com>

Dependencies:
- [x] https://github.com/kernelci/test-definitions/pull/7
- [x] #603 
- [x] #599

### CODE REVIEW

**Comment by Lakshmipathi** [comment]
depends on https://github.com/kernelci/kernelci-core/pull/603

**Comment by Lakshmipathi** [comment]
Sample Results from staging:
https://staging.kernelci.org/test/job/kernelci/branch/staging-mainline/kernel/staging-mainline-20210416.0/plan/ltp-timers/
https://staging.kernelci.org/test/plan/id/6078e0a20bfe3738d87747c2/
https://staging.kernelci.org/test/plan/id/6078df4f4ca8b9707a7747ef/

**Review by gctucker — APPROVED** [review]

File: config/core/test-configs.yaml [hunk 1]
```diff
@@ -1616,7 +1624,6 @@ test_configs:
       - baseline
       - ltp-crypto
       - ltp-ipc
-      - ltp-mm
```
Comment (gctucker) [inline config/core/test-configs.yaml:1619]: Please move this to a separate PR, it's true we should stop running `ltp-mm` on `bcm2836-rpi-2-b` because it's taking too long to complete.  But it's not related to this commit about timers.

**Review by gctucker — CHANGES_REQUESTED** [review]

**Review by gctucker — APPROVED** [review]
